astropy/stats/jackknife.py

```diff
@@ -116,11 +116,11 @@ def jackknife_stats(data, statistic, confidence_level=0.95):
     >>> estimate, bias, stderr, conf_interval = jackknife_stats(
     ...     data, test_statistic, 0.95)
     >>> estimate
-    4.5
+    np.float64(4.5)
     >>> bias
-    0.0
+    np.float64(0.0)
```

Comment:
I'm confused, I thought a few years ago we were told numpy's float64 was an alias for the pure Python float? In any case I wonder if we should consider whether what we actually want to do is return numpy scalars or Python scalars now that they look different?

Reply:
> I thought a few years ago we were told numpy's float64 was an alias for the pure Python float?

@astrofrog maybe you got this from Numpy 1.20's release notes about deprecated type alias ?
https://numpy.org/doc/stable/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated

Also note that [Python's doc](https://docs.python.org/3/tutorial/floatingpoint.html#representation-error) mentions
> and almost all platforms map Python floats to IEEE 754 binary64 “double precision” values

so numpy's `float64` cannot be aliased to `float` on all platforms.

Reply:
It is indeed not the same - it has `.dtype`, etc. Indeed, this is why the change was made - to make clear to the user that they are not identical -- which of course is more useful for single precision, different integers, etc.

We could indeed reconsider what we return, though I think in the end it does not matter much.